duckduckgo/iOS
# update Device.swift

Task/Issue URL: https://app.asana.com/0/392891325557410/624047251318055
Tech Design URL: n/a
CC:

**Description**:

Update to latest version of Device.swift and update IDE settings for Xcode 9.3

**Steps to test this PR**:

1. Update Carthage 
1. Run the app on the simulator and browse to a page, should not crash
1. Run the app on a device and browse to a page, should not crash
1. On a device with mail configured, create feedback email and ensure device is specified correctly


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
